R/commonsummarystatsttestbayesian.R

```diff
@@ -361,7 +361,7 @@
     n2                        = 0 ,
     paired                    = FALSE,
     BF10user                  = BFlist[["BF10"]],
-    yAxisLegendRobustnessPlot = BFPlots,
+    yAxisLegendRobustnessPlot = BFPlotsH1H0,
```

Comment:
@JTPetter Thanks for figuring this out!

@vandenman It seems that this plots H1 above and H0 below if TRUE, whereas it plots H0 above and H1 below if FALSE. Did something change in the development process?

Reply:
nothing changed to the JASPgraphs code in 5 months, so I don't think so. Perhaps this was introduced during the rewrite?